pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceUsageConsumer.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.resourcegroup;
+
+import org.apache.pulsar.broker.service.resource.usage.ResourceUsage;
+
+/*
+ * Interface that a resource owner (tenant, namespace or topic) needs to implement.
+ */
+public interface ResourceUsageConsumer {
+    /*
+     * Get the unique identifier for the owner
+     * @return return the owner ID
+     */
+    String getID();
+
+    /*
+     * Listener for resource usage published by other brokers
+     * @param broker name
+     * @param Resource usage object
+     */
```

Comment:
What about 'acceptResourceUsage'?

Reply:
@eolivelli , I changed the name of the method, per your suggestion. Please take a look.

Reply:
@eolivelli I have incorporated your comments and the CI passed as well. Can you please take a look and approve. Thank you!

Reply:
good, thanks